python/ray/tests/test_advanced_9.py

```diff
@@ -438,6 +437,42 @@ def f():
         assert ray.get(f.remote())
 
 
+def test_gcs_fd_usage(shutdown_only):
+    ray.init(
+        _system_config={
+            "prestart_worker_first_driver": False,
+            "enable_worker_prestart": False,
+        },
+    )
+    gcs_process = ray._private.worker._global_node.all_processes["gcs_server"][0]
+    gcs_process = psutil.Process(gcs_process.process.pid)
+    print("GCS connections", len(gcs_process.connections()))
+
+    @ray.remote(runtime_env={"env_vars": {"Hello": "World"}})
+    class A:
+        def f(self):
+            import os
+
+            return os.environ.get("Hello")
+
+    # In case there are still some pre-start workers, consume all of them
```

Comment:
Is this comment still relevant? Looks like you turned off the feature.

Regardless, I probably prefer to test with the default option with this warm up.

Reply:
I think here, even it's turned off, somehow, there are still some got pre-started. It's not going with this code path I believe.

This test focus on socket usages, so I don't think it's worth testing with default options.